e2e/run-programmatically-multiple-projects/run-jest.js

```diff
@@ -5,18 +5,20 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-const {runCLI} = require('@jest/core');
+const {createJest} = require('jest');
```

Comment:
instead of changing this test, can you add a new one?

Reply:
@nicojs missed?

Reply:
I've moved the old one to "run-cli.js", so this project now has 2 entry files.